stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -1087,6 +1088,17 @@ public struct Data : ReferenceConvertible, Equatable, Hashable, RandomAccessColl
         _sliceRange = range
     }
     
+    @inline(__always)
+    func _validateIndex(_ index: Int, message: String? = nil) {
```

Comment:
Hmm, it looks like `_validateIndex` is never actually called with a message. We can probably actually omit it...